README.md

```diff
@@ -38,4 +38,4 @@ For example, to translate the Christopher Columbus in Richmond post, copy `conte
 If the language is not supported, e.g. German, then create `content/deutsch/post/columbus-rva.md` and translate the post. Then, update the `config.toml` file in the root directory to add deutsch support.
 
 ### Categories and tags
-The categories inside each post can include one or more of the following: `["colonizers", "confederates", "racists", "rapists", "slavers", "war-criminals"]` and the following tags can include one or more of the following: `["municipal-action", "renaming", "direct-action", "private-action"]`. Other tags and categories can be added, but first raise an issue about it.
+The categories inside each post can include one or more of the following: `["colonizers", "confederates", "racists", "rapists", "slavers", "war-criminals"]` and the following tags can include one or more of the following: `["municipal-action", "renaming", "direct-action", "private-action"]`. Other tags and categories can be added, but first raise an issue about it.  You should also set the subject as the name of the person who is depicted by the statue.
```

Comment:
This should clarify it's an array, since many of these entries are about one person, and could reasonably be a stand-alone string, rather than a string inside an array.

This is great work otherwise and major improvement!

Reply:
Clarified the language.  Ideally we can depend on the template in #92 to help people know how to format it.